cpp/src/io/functions.cpp

```diff
@@ -470,8 +470,8 @@ void orc_chunked_writer::close()
   writer->close();
 }
 
-using namespace cudf::io::detail::parquet;
-namespace detail_parquet = cudf::io::detail::parquet;
+using namespace cudf::io::parquet::detail;
+namespace detail_parquet = cudf::io::parquet::detail;
```

Comment:
I think this only exists to disambiguate between reader/writer data members and reader/writer classes. Kind of odd, IMO. Maybe a battle for another day.